rcl/Doxyfile

```diff
@@ -0,0 +1,17 @@
+# All settings not listed here will use the Doxygen default values.
+
+PROJECT_NAME           = "rcl"
+PROJECT_NUMBER         = beta1
```

Comment:
It's the version number, it has to be updated every time we change the version. What should I put there instead?